resources.go

```diff
@@ -93,14 +93,6 @@ func Provider() tfbridge.ProviderInfo {
 			// Map each resource in the Terraform provider to a Pulumi type. An example
 			// is below.
 			"packet_bgp_session": {Tok: makeResource(mainMod, "BgpSession")},
-			"packet_connect": {
-				Tok: makeResource(mainMod, "Connect"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"facility": {
-						Type: makeType(mainMod, "Facility"),
-					},
-				},
-			},
```

Comment:
What happened to this resource? Also:
1. Should we delete the corresponding files in each SDK?
2. Is this a breaking change?

Reply:
They were actually remove in 1.4.0 - this just cleans up the error on the `make` step where the resource isn't in the provider list. No files in the SDK were changed related to this release